hack/ci/install-cnv.sh

```diff
@@ -2,12 +2,45 @@
 
 set -ex
 
+CNV_VERSION=4.12
```

Comment:
we should to maintain two paths here. One for installing from the official stable channel and one for installing a pre-release.

Here's an idea of how we could do this with 3 env vars.

**CNV_PRERELEASE_VERSION**: This would be a rename of `CNV_VERSION` to `CNV_PRERELEASE_VERSION` and have it default to being empty (not set). 
**CNV_RELEASE_CHANNEL**: defaults to "stable" but overrides to nightly-${CNV_PRERELEASEVERSION} when a prerelease is used.
**CNV_SUBSCRIPTION_SOURCE**: defaults to `redhat-operators` but is overwritten to `cnv-nightly-catalog-source` when CNV_PRERELEASE_VERSION is set


The end result here is that all we have to do to enable usage of a pre-release is set the CNV_PRERELEASE_VERSION=4.12 env var before calling the install-cnv.sh script. If that ENV VAR isn't set, then we get the default stable version.

Would this make sense?